pkg/build/builder/daemonless.go

```diff
@@ -306,10 +306,43 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 func generateTransientMounts() []string {
 	mounts := []string{}
 	mounts = appendRHSMMount(mounts)
+	mounts = appendETCPKIMount(mounts)
```

Comment:
We also need to mount the `redhat.repo` file. This is only one file.

Reply:
right,  but how to handle a top level file vs. a dir with the buildah temporary mount options is not obvious to me @adambkaplan @nalind since all the man pages / info that I can find only talks in the context of dirs

or can I ultimate create an 

"<tmpdir>/redhat.repo:/run/secrets/redhat.repo:rw,nodev,noexec,nosuid"

entry in the array of strings fed into the buildah transient mounts?

Reply:
That should work, but processes that try to modify it may get confused by it being a bind mount - it can't be removed, and nothing can be renamed on top of it.

Reply:
thanks @nalind 

I *think* that should be OK

Or am I forgetting something @adambkaplan ?

I'll cook up the change in the interim

Reply:
update pushed

Reply:
I am a little curious how our e2e that does the recursive chmod would do if we had host entitlements in CI

I'm going to update the BZ to make sure QE covers this as well